### ISSUE TITLE
NoSuchMethodError

### ISSUE THREAD

**Issue by Pitel** [pbreault/adb-idea#26]
Plugin version 1.2.2, Android Studio 1.5 (canary), multiple modules, multiple devices.

When I want to do something with your plugin (which is awesome, btw) I get the dialog which let me choose module (app or wear, in my project) and when I choose one, it crashes with the following stack trace:
```
com.android.tools.idea.model.ManifestInfo.getRequiredFeatures()Ljava/util/List;
java.lang.NoSuchMethodError: com.android.tools.idea.model.ManifestInfo.getRequiredFeatures()Ljava/util/List;
	at com.developerphil.adbidea.ui.MyDeviceChooser.<init>(MyDeviceChooser.java:117)
	at com.developerphil.adbidea.ui.DeviceChooserDialog.<init>(DeviceChooserDialog.java:36)
	at com.developerphil.adbidea.adb.AdbFacade.askUserForDevice(AdbFacade.java:119)
	at com.developerphil.adbidea.adb.AdbFacade.getDevice(AdbFacade.java:91)
	at com.developerphil.adbidea.adb.AdbFacade.executeOnDevice(AdbFacade.java:51)
	at com.developerphil.adbidea.adb.AdbFacade.clearDataAndRestart(AdbFacade.java:47)
	at com.developerphil.adbidea.action.ClearDataAndRestartAction.actionPerformed(ClearDataAndRestartAction.java:10)
	at com.developerphil.adbidea.action.AdbAction.actionPerformed(AdbAction.java:13)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:182)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter$1.run(ActionMenuItem.java:312)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:958)
	at com.intellij.openapi.wm.impl.IdeFocusManagerImpl.runOnOwnContext(IdeFocusManagerImpl.java:124)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.actionPerformed(ActionMenuItem.java:282)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.java:109)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:512)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.access$300(BegMenuItemUI.java:44)
	at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:532)
	at java.awt.Component.processMouseEvent(Component.java:6525)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6290)
	at java.awt.Container.processEvent(Container.java:2234)
	at java.awt.Component.dispatchEventImpl(Component.java:4881)
	at java.awt.Container.dispatchEventImpl(Container.java:2292)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4898)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4533)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4462)
	at java.awt.Container.dispatchEventImpl(Container.java:2278)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:734)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:565)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by Pitel** [comment]
It's probably already fixed here, right? a102ad598619a48b7052a8c0484ec3717d23c0eb

**Comment by pbreault** [comment]
Hi @Pitel, some api changed in 1.5.
I'm working on a fix.
Thanks!

**Comment by pbreault** [comment]
This is fixed in 1.2.3.
Click on "Check for Updates" and everything should be fine!

**Comment by Pitel** [comment]
Yup, it works, thanks. :+1:
